controller/securenaming.go

```diff
@@ -41,15 +41,15 @@ func NewSecureNamingController(core v1core.CoreV1Interface) *SecureNamingControl
 	return &SecureNamingController{sc, si}
 }
 
-func newServiceIndexerInformer(core v1core.CoreV1Interface) (cache.Indexer, *cache.Controller) {
```

Comment:
I'd prefer to add comments in a separate commit since it's unrelated to this commit.

I've create issue #19 to track that.